site/src/docs/patterns/cookies/common-helsinki-cookies.mdx

```diff
@@ -28,7 +28,7 @@ import ExternalLink from '../../../components/ExternalLink';
 
 ## Common cookies between \*.hel.fi sites
 
-The following is a list of approved common cookies between services using \*.hel.fi domain. You can find this list as a JSON object in the <ExternalLink href="https://github.com/City-of-Helsinki/helsinki-design-system/blob/master/packages/react/src/components/cookieConsent/content.json" openInNewTab openInNewTabAriaLabel="Opens in a new tab" openInExternalDomainAriaLabel="Opens a different website">Helsinki Design System GitHub repository</ExternalLink>.
+The following is a list of approved common cookies between services using \*.hel.fi domain. You can find this list as a JSON object in the <ExternalLink href="https://github.com/City-of-Helsinki/helsinki-design-system/blob/master/packages/react/src/components/cookieConsent/getContent.ts" openInNewTab openInNewTabAriaLabel="Opens in a new tab" openInExternalDomainAriaLabel="Opens a different website">Helsinki Design System GitHub repository</ExternalLink>.
```

Comment:
This is actually getting fixed also in Niko's [cookie PR ](https://github.com/City-of-Helsinki/helsinki-design-system/pull/943)

Reply:
Nice catch! I might remove this so this doesn't cause an unnecessary conflict while merging. I'm not sure if it would but it's no biggie.